xmake/plugins/project/clang/compile_commands.lua

```diff
@@ -44,8 +44,10 @@ function _translate_arguments(arguments)
         if idx == 1 and is_host("windows") and path.extension(arg) == "" then
             arg = arg .. ".exe"
         end
-        if arg:find("-isystem", 1, true) then
-            arg = arg:replace("-isystem", "-I")
+        if arg:startswith("-isystem-after", 1, true) then
+            arg = "-I" .. arg:sub(13, -1)
```

Comment:
why? startswith? len is 12?  `#("-isystem-after")` is 14

Reply:
Oh, my mistake you right !

Reply:
Resolved in [91a84f7](https://github.com/xmake-io/xmake/pull/2668/commits/91a84f7536e1563b448e85e2c4dacc70bcdd7a82)